boards/nucleo144-f207/Makefile.include

```diff
@@ -2,5 +2,8 @@
 export CPU = stm32f2
 export CPU_MODEL = stm32f207zg
 
-# load the common Makefile.include for Nucleo boards
-include $(RIOTBOARD)/nucleo-common/Makefile.include
+# include nucleo common serial configuration
+include $(RIOTBOARD)/nucleo-common/Makefile.include.serial
```

Comment:
looking at my changes again, this line is not required as it's already include by indirectly by the line above. I'll update this for others.